release/ray_release/byod/byod_rllib_contrib_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image 
+# to run the agent stress test.
+
+set -exo pipefail
+
+
+# TODO: https://github.com/Farama-Foundation/AutoROM/issues/48
+pip3 install https://ray-ci-deps-wheels.s3.us-west-2.amazonaws.com/AutoROM.accept_rom_license-0.5.4-py3-none-any.whl
+#git clone https://github.com/ray-project/rl-experiments.git
+unzip rl-experiments/halfcheetah-sac/2022-12-17/halfcheetah_1500_mean_reward_sac.zip -d ~/.
+pip3 uninstall -y minigrid
+#pip3 install torch==2.0.0+cu118 torchvision==0.15.1+cu118 --index-url https://download.pytorch.org/whl/cu118
+
+# Clone the repo in order to be able to install individual contrib packages.
+git clone https://github.com/ray-project/ray.git
```

Comment:
Yes, this is intended. The reason is that the only way to run these `rllib_contrib` algos is by cloning the Ray repo, cd'ing into `rllib_contrib`, installing the required dependencies (including some pinned version of Ray), and then run the example scripts.

Reply:
then could you also checkout the same commit, rather than using master head?

Reply:
what we want is that, if the code of the commit that triggers the test does not change, the build/test result won't change. otherwise, we would not be able to bisect the error.